pkg/types/validation/installconfig.go

```diff
@@ -19,9 +19,19 @@ import (
 	"github.com/openshift/installer/pkg/validate"
 )
 
+const (
+	installConfigVersion = "v1beta1"
```

Comment:
Do we want to SemVer this to make it easier to see feature additions?  Or will we not bump on feature additions and only bunp this for backwards-incompatible changes?

Reply:
I was originally going to use semver, but then decided with K8S versioning. This was just done for continuity with the rest of the ecosystem. If we were to adopt semver instead, would you still want to use the same field, or should we create a different version field?

Reply:
> I was originally going to use semver, but then decided with K8S versioning.

That's fine with me, I just want to know what the plan is ;).

> This was just done for continuity with the rest of the ecosystem. If we were to adopt semver instead, would you still want to use the same field, or should we create a different version field?

I'm fine using this field for whatever version strings we pick.